swift/Workflow/Sources/SubtreeManager.swift

```diff
@@ -419,7 +445,7 @@ extension WorkflowNode.SubtreeManager {
                 .observe(on: QueueScheduler.workflowExecution)
                 .start { [weak self] event in
                     switch event {
-                    case .value(let output):
+                    case let .value(output):
```

Comment:
nit: I prefer the `let` inside the case for the reason outlined in the [Google Swift style guide](https://google.github.io/swift/#pattern-matching)

Reply:
This will require a `swiftformat` rules change, will do it in a subsequent PR.